drivers/cleanup.py

```diff
@@ -3103,8 +3103,7 @@ def __init__(self, srUuid):
         self._srLock = lock.Lock(vhdutil.LOCK_TYPE_SR, srUuid)
 
     def acquireNoblock(self):
```

Comment:
The name of this method is now a bit misleading, although I can't think of a pith, accurate replacement

Reply:
No, it's not as it is the API method for the super-class of this object type, that being lock.Lock and we are doing an acquireNoblock on the GC active lock, with the extra caveat that this needs the SR lock before it can check the GC active lock.